client/views/createWishcard.ejs

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Profile Page | Donate Gifts</title>
+
+  <%- include('./templates/header')-%>
+
+  <link rel="stylesheet" href="/public/css/profile.css" />
+  <link rel="stylesheet" href="/public/css/about.css" />
+
+  <!-- CDN jQuery -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+  <%- include('./templates/nav')-%> <%- include('./templates/editAboutMeModal')-%>
+  <!-- header section-->
+ 
+  <% if (user.userRole == 'partner' && locals.agency && agency.isVerified) { %>
```

Comment:
Does it redirect to 400 page for the ELSE case here?

Reply:
You are right it is not necessary here since on the server we validate if it's an agency user. Will remove it